### PR TITLE
Fixes #192 Permissions required by setup-sa.sh not documented

### LINKED ISSUE #192: Permissions required by setup-sa.sh not documented
I have a user that does not have the correct permissions to run the setup-sa.sh script.  The documentation does not tell me what permissions I need in order to run this scripts.  Here is my error:

```console
 $ ./helpers/setup-sa.sh my-org my-project
Verifying organization...
Verifying project...
Skipping billing account verification... (parameter not passed)
Creating Seed Service Account...
Created service account [project-factory-6999].
Downloading key to credentials.json...
Applying permissions for org org-number and project chlove-baseproject...
ERROR: (gcloud.organizations.add-iam-policy-binding) User [chlove@google.com] does not have permission to access organization [org-number:getIamPolicy] (or it may not exist): The caller does not have permission
```

I removed org number and project id for security sake.

What permissions does the user need to run the setup-sa.sh script?

### PR DESCRIPTION
**Pull request by ivankorn**
#192 
- Added minimum permissions required to run setup.sh into README.MD
- Auto-formatted README.MD
- Added billing account to the list of arguments for setup.sh into README.MD

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -188,9 +188,26 @@ Account in the Seed Project, grant the necessary roles to the Seed Service
 Account, and enable the necessary API's in the Seed Project.  Run it as follows:
 
 ```sh
-./helpers/setup-sa.sh <ORGANIZATION_ID> <SEED_PROJECT_NAME>
+./helpers/setup-sa.sh <ORGANIZATION_ID> <SEED_PROJECT_NAME> <BILLING_ACCOUNT>
```
Comment (aaron-lane) [inline README.md]: We should indicate that the billing account is optional.

```suggestion
./helpers/setup-sa.sh <ORGANIZATION_ID> <SEED_PROJECT_NAME> [BILLING_ACCOUNT]
```

File: README.md [hunk 2]
```diff
@@ -188,9 +188,26 @@ Account in the Seed Project, grant the necessary roles to the Seed Service
...
 ```
 
+In order to execute this script you must have an account with following list of
```
Comment (aaron-lane) [inline README.md]:
```suggestion
In order to execute this script, you must have an account with the following list of
```

File: README.md [hunk 3]
```diff
@@ -188,9 +188,26 @@ Account in the Seed Project, grant the necessary roles to the Seed Service
...
+bare-minimum permissions:
```
Comment (aaron-lane) [inline README.md]:
```suggestion
permissions:
```

File: README.md [hunk 4]
```diff
@@ -104,51 +104,51 @@ determining that location is as follows:
...
-| org\_id | The organization ID. | string | n/a | yes |
-| project\_id | If provided, the project uses the given project ID. Mutually exclusive with random_project_id being true. | string | `""` | no |
-| random\_project\_id | Enables project random id generation. Mutually exclusive with project_id being non-empty. | string | `"false"` | no |
-| sa\_role | A role to give the default Service Account for the project (defaults to none) | string | `""` | no |
-| shared\_vpc | The ID of the host project which hosts the shared VPC | string | `""` | no |
-| shared\_vpc\_subnets | List of subnets fully qualified subnet IDs (ie. projects/$project_id/regions/$region/subnetworks/$subnet_id) | list | `<list>` | no |
-| usage\_bucket\_name | Name of a GCS bucket to store GCE usage reports in (optional) | string | `""` | no |
-| usage\_bucket\_prefix | Prefix in the GCS bucket to store GCE usage reports in (optional) | string | `""` | no |
+| Name                           | Description                                                                                                                                                            |     Type     |     Default      | Required |
```
Comment (aaron-lane) [inline README.md]: Can you please undo this formatting? The Inputs and Outputs tables should remain as generated by `terraform-docs`.

File: README.md [hunk 5]
```diff
@@ -188,9 +188,26 @@ Account in the Seed Project, grant the necessary roles to the Seed Service
...
 
 ```sh
-./helpers/setup-sa.sh <ORGANIZATION_ID> <SEED_PROJECT_NAME>
+./helpers/setup-sa.sh <ORGANIZATION_ID> <SEED_PROJECT_NAME> <BILLING_ACCOUNT>
 ```
 
+In order to execute this script you must have an account with following list of
+bare-minimum permissions:
+
+- `resourcemanager.organizations.list`
```
Comment (aaron-lane) [inline README.md]: Are we missing `resourcemanager.projects.list` and `billing.accounts.list`?

File: README.md [hunk 6]
```diff
@@ -188,9 +188,26 @@ Account in the Seed Project, grant the necessary roles to the Seed Service
...
+- `resourcemanager.organizations.setIamPolicy`
+- `iam.serviceAccounts.create`
+- `iam.serviceAccountKeys.create`
+- `resourcemanager.projects.setIamPolicy`
+- `serviceusage.services.enable` on the project
+- `servicemanagement.services.bind` on following services:
```
Comment (aaron-lane) [inline README.md]: Are we missing `cloudresourcemanager.googleapis.com`?

File: README.md (hunk 4 above)
Comment (ivankorn) [reply README.md]: Sure, with next commit

Comment (ivankorn) [reply README.md]: @aaron-lane, just fixed this one in [e41724f](https://github.com/terraform-google-modules/terraform-google-project-factory/pull/230/commits/e41724ff27932ad84dc65faf8e6c4d3307beff5e)

File: README.md (hunk 6 above)
Comment (ivankorn) [reply README.md]: Yes! Fixed that in [5e878a1](https://github.com/terraform-google-modules/terraform-google-project-factory/pull/230/commits/5e878a134a8d42dc5405439541013515fa372562)
Thank you!

File: README.md (hunk 5 above)
Comment (ivankorn) [reply README.md]: Yes!
Fixed that in [5e878a1](https://github.com/terraform-google-modules/terraform-google-project-factory/pull/230/commits/5e878a134a8d42dc5405439541013515fa372562)
Thank you for great review, @aaron-lane! Good eye!

**Pull request merged**